pallets/crowdloan/src/tests.rs

```diff
@@ -0,0 +1,219 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2021 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use frame_support::{assert_noop, assert_ok};
+use sp_runtime::traits::BadOrigin;
+
+use crate::{mock::*, Error};
+
+// set_admin_account
+
+#[test]
+fn test_set_admin_account() {
+	let admin = ACCOUNT_00;
+	let new_admin = ACCOUNT_01;
+
+	ExtBuilder::default()
+		.with_admin_account(admin.clone())
+		.build()
+		.execute_with(|| {
+			assert_eq!(Crowdloan::admin_account(), admin);
+
+			// Change admin
+			assert_ok!(Crowdloan::set_admin_account(
+				Origin::signed(admin.clone()),
+				new_admin.clone()
+			));
+
+			// Test new admin is the one set
+			assert_eq!(Crowdloan::admin_account(), new_admin);
+		});
+}
+
+#[test]
+fn test_set_admin_account_with_sudo() {
+	let admin = ACCOUNT_00;
+	let new_admin = ACCOUNT_01;
+
+	ExtBuilder::default()
+		.with_admin_account(admin.clone())
+		.build()
+		.execute_with(|| {
+			assert_eq!(Crowdloan::admin_account(), admin);
+
+			// Change admin with sudo account
+			assert_ok!(Crowdloan::set_admin_account(Origin::root(), new_admin.clone()));
+
+			// Test new admin is the one set
+			assert_eq!(Crowdloan::admin_account(), new_admin);
+		});
+}
+
+#[test]
+fn test_no_admin_set() {
```

Comment:
The test name confused me so I had to go back to the mock to check the default admin setting.

It appears that the derived `default` implementation for the admin account is indeed `ACCOUNT_00`? Else the first assert would fail. Not sure what to think about this hidden magic. In the end, it's just a test and does not matter too much. Still wanted to raise this.

W: Rename it.
```suggestion
fn test_no_custom_admin_set {
```